dom-event-delegation/main.js

```diff
@@ -0,0 +1,15 @@
+const $taskList = document.querySelector('.task-list');
+
+$taskList.addEventListener('click', (event) => {
+  console.log(
+    'event target: ',
+    event.target,
+    'event target tag name: ',
+    event.target.tagName
+  );
+  if (event.target && event.target.tagName === 'BUTTON') {
+    const taskListItem = event.target.closest('.task-list-item');
```

Comment:
Since you're storing a DOM element, you should put the `$` in front of the variable name.